content/en/bits_ai/mcp_server/setup/_index.md

```diff
@@ -0,0 +1,331 @@
+| [Codex CLI](https://help.openai.com/en/articles/11096431-openai-codex-cli-getting-started) | OpenAI | |
+| [Cursor](https://www.cursor.com/) | Anysphere | Datadog [VS Code & Cursor extension](#connect-in-vs-code-or-cursor) recommended. |
+| [Goose](https://github.com/block/goose) | Block | |
+| [Q CLI](https://github.com/aws/amazon-q-developer-cli) | Amazon | Does not support HTTP transport for remote authentication. Use [local binary authentication](#connect-to-the-datadog-mcp-server) instead. |
+| [VS Code](https://code.visualstudio.com/) | Microsoft | Datadog [VS Code & Cursor extension](#connect-in-vs-code-or-cursor) recommended. |
+
+### Connect in VS Code or Cursor
+
+Datadog's [VS Code and Cursor extension][1] includes built-in access to the Datadog MCP Server. Benefits include:
+
+* No additional MCP Server setup after you install the extension and connect to Datadog.
+* One-click transitions between multiple Datadog organizations.
+* Better fixes from **Fix in Chat** on Code Insights (issues from Error Tracking, Code Vulnerabilities, and Library Vulnerabilities), informed by context from the MCP Server.
+
+To install the extension:
+
+1. Ensure you have an active version of [Node.js][2] installed on your local machine.  
+2. If you previously installed the Datadog MCP Server manually, remove it from the IDE's configuration to avoid conflicts. To find the MCP Server configuration:
+   - VS Code: Open the command palette (`Shift` + `Cmd/Ctrl` + `P`) and run `MCP: Open User Configuration`.
+   - Cursor: Go to **Cursor Settings** (`Shift` + `Cmd/Ctrl` + `J`) and select the **Tools & Integrations** tab.
+3. Install the Datadog extension following [these instructions][3]. If you have the extension installed already, make sure it's the latest version, as new features are released regularly.  
+4. Sign in to your Datadog account. If you have multiple accounts, use the account included in your Product Preview.
+    {{< img src="bits_ai/mcp_server/ide_sign_in.png" alt="Sign in to Datadog from the IDE extension" style="width:70%;" >}}
+5. Open a project or workspace.
+6. **Restart the IDE.**
+7. Confirm the Datadog MCP Server is available and the [tools](#available-tools) are listed in your IDE:
+    - VS Code: Open the chat panel, select Agent mode, and click the **Configure Tools** button.
+       {{< img src="bits_ai/mcp_server/vscode_configure_tools_button.png" alt="Configure Tools button in VS Code" style="width:70%;" >}}
+    - Cursor: Go to **Cursor Settings** (`Shift` + `Cmd/Ctrl` + `J`), and select the **Tools & Integrations** tab.
+
+## Available tools
+
+This section lists the tools available in the Datadog MCP Server and provides example prompts for using them.
+
+<div class="alert alert-warning">Datadog MCP Server tools are under significant development and are subject to change. We encourage you to share any feedback, use cases, or issues encountered with your prompts and queries.</div>
```

Comment:
Why is this callout a warning, when the previous one was info? Also, please reword this to avoid the first person pronoun.